rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -73,11 +109,47 @@ def test_paraswap_swap_with_fee(accountant: 'Accountant'):
         pot.events_accountant.process(event=event, events_iterator=events_iterator)  # type: ignore
 
     extra_data = {
-        'group_id': '1' + tx_hash.hex() + '12',  # pylint: disable=no-member
+        'group_id': '1' + tx_hash.hex() + '23',  # pylint: disable=no-member
         'tx_hash': tx_hash.hex(),  # pylint: disable=no-member
     }
+    aquisition_event = AssetAcquisitionEvent(
+        index=0,
+        timestamp=TIMESTAMP_0_SEC,
+        amount=FVal(swap_amount_str),
+        rate=Price(FVal(0.5)),
+    )
+    aquisition_event.remaining_amount = ZERO  # since it's not settable at ctor
     expected_processed_events = [
         ProcessedAccountingEvent(
+            index=0,
+            event_type=AccountingEventType.TRANSACTION_EVENT,
+            notes=f'Receive {swap_amount_str} WBTC from {aquired_from_address} to {user_address}',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_0_SEC,
+            asset=A_WBTC,
+            free_amount=ZERO,
+            taxable_amount=FVal(swap_amount_str),
+            price=Price(FVal(0.5)),
+            pnl=PNL(taxable=FVal(swap_amount_str) / 2, free=ZERO),
+            cost_basis=None,
+            extra_data={'tx_hash': tx_hash.hex()},  # pylint: disable=no-member
+        ),
+        ProcessedAccountingEvent(
+            index=1,
+            event_type=AccountingEventType.TRANSACTION_EVENT,
+            notes=f'Receive {ONE} USDC from {aquired_from_address} to {user_address}',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_0_SEC,
+            asset=A_USDC,
+            free_amount=ZERO,
+            taxable_amount=FVal(ONE),
```

Comment:
here and in the other places ONE is already an Fval. You don't need to FVal it again